src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,32 +164,136 @@ export function EmbeddedPositionSearchPanel({
+            </Stack>
+        );
+    };
+
     return (
         <div key='targetGroup'
-             style={{display: 'flex', flexDirection: 'column', alignItems: 'center', alignSelf: 'stretch', height:'100%',
-                 paddingBottom:insetSpacial?0:20, position: 'relative'}}>
-            <HiPSTargetView {...{
+             onMouseDown={() => {
+                      if (!isSearchPanel) setIsHovered(false);
+                      else setIsHovered(true);
+             }}
+            style={{display: 'flex', flexDirection: 'column', alignItems: 'center', alignSelf: 'stretch', height:'100%',
+                         paddingBottom:insetSpacial?0:20, position: 'relative'}}>
+            <HiPSTargetView
+                {...{
                 hipsUrl, centerPt:initCenterPt, hipsFOVInDeg, mocList, coordinateSys, sRegion, plotId,
                 minSize: minValue, maxSize: maxValue, toolbarHelpId,
                 whichOverlay: doGetConeAreaOp(), setWhichOverlay: doToggle ? setConeAreaOp : undefined,
                 targetKey, sizeKey, polygonKey, sx: {minHeight: 300, alignSelf: 'stretch', flexGrow:1}
             }}/>
-            <Sheet {...{className:`FFepsp-content ${insetSpacial ? 'inset' : ''}`,
-                sx: (theme) => (
-                    {
-                        alignItems: 'center',
-                        alignSelf: 'stretch',
-                        borderRadius: '5px 5px 2px 2px',
-                        border: `3px solid ${theme.vars.palette['neutral']?.softActiveBg}`,
-                        position: 'absolute',
-                        px: 1/2,
-                        bottom: '1.5rem',
-                        left: 3,
-                        ...slotProps?.searchRoot?.sx
-                    })
-
-            }}>
-                {wrappedInternals}
+            <Sheet
+                onMouseEnter={() => {
+                    setIsHovered(true);
+                    setIsSearchPanel(true);
+                }}
+                onMouseLeave={() => {
+                    setIsSearchPanel(false);
+                }}
+                {...{className:`FFepsp-content ${insetSpacial ? 'inset' : ''}`,
+                    sx: (theme) => (
+                        {
+                            alignItems: 'center',
+                            alignSelf: 'stretch',
+                            borderRadius: '5px 5px 2px 2px',
+                            border: `3px solid ${theme.vars.palette['neutral']?.softActiveBg}`,
+                            position: 'absolute',
+                            px: 1/2,
+                            bottom: '1.5rem',
+                            maxWidth: '50em',
+                            left: 3,
+                            opacity: isHovered ? '100%' : '40%',
+                            ...slotProps?.searchRoot?.sx
+                        })
+                }}>
+                <CollapsibleGroup
+                    sx={{borderWidth: '0px',
```

Comment:
CollapsibleGroup uses `variant='outlined'` by default.  To remove border, you should set a different variant='plain'.